File: src/7.ts
import primes from './core/primes';

// Computes the nth prime number.
//
// @remarks
//
// Problem cheats.  Just uses a precomputed list of prime numbers.
//
// See {@link https://projecteuler.net/problem=7}.
export function compute(n: number) {
  return primes()[n - 1];
}

File: project_euler/7.py
# Problem 7: 10001st Prime
#
# What is the 10001st prime number?
#
# See https://projecteuler.net/problem=7
from sympy import prime


def run(n=10_001) -> int:
    return int(prime(n))


def test_run():
    assert run() == 104743
